<launch>
  <arg name="model" default="$(find wpr1_bringup)/urdf/wpr1.urdf"/>
  <param name="robot_description" command="$(find xacro)/xacro.py $(arg model)" />
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />

  <!-- wpr1 manipulator-->
  <node pkg="wpr1_bringup" type="wpr1_manipulator" name="wpr1_manipulator" output="screen">
    <param name="serial_port" type="string" value="/dev/wpr1_mani"/> 
  </node>

  <!--- Run Kinect -->
  <include file="$(find kinect2_bridge)/launch/kinect2_bridge.launch" />

  <!-- box_capture_node -->
  <node name="box_capture_node" pkg="wpr_rack_pkg" type="box_capture_node" output="screen"/>

  <!--- RViz  -->
  <arg name="rvizconfig" default="$(find wpr_rack_pkg)/rviz/box_capture.rviz" />
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" />

</launch>
